### DISCUSSION TITLE
Chance on Hit/Use

### DISCUSSION THREAD

**Discussion by WehrJ** [AscensionGameDev/Intersect-Engine#1535] [Ideas]
I would like to be able to create items that have a % chance at proc'ing an action. If we could do this one equipment and consumables, we could create unique consumables and equipment.

Add a "Chance on..." Selection  on equipment and consumables including: 
- being hit
-hitting target 
- Kill
- Death
- Use

Add a % chance

Add a selection of what happens: 
- Spell
- Projectile
- Event

**Additional context**

**Comment by WeylonSantana** [comment]
I think it's very specific, I'm against

**Comment by SilverShardOnline** [comment]
I like the idea, its a very classic MMORPG feature.
It maybe too specific, but also so are many features.
